cmd/contour/serve.go

```diff
@@ -399,6 +405,9 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 		}
 	}
 
+	// Set up workgroup runner and register informers.
+	var g workgroup.Group
+
 	// Only inform on GatewayAPI resources if Gateway API is found.
 	if ctx.Config.GatewayConfig != nil {
 		if clients.ResourcesExist(k8s.GatewayAPIResources()...) {
```

Comment:
we're still setting up the informers for Gateway API resources too, is that intended?